webrtc.html

```diff
@@ -8777,13 +8775,25 @@ <h2 id="sec.stats-model">Statistics Model</h2>
               MUST run the following steps:</p>
               <ol>
                 <li>
-                  <p>Let <var>selectorArg</var> be the methods first
+                  <p>Let <var>selectorArg</var> be the method's first
                   argument.</p>
                 </li>
                 <li>
-                  <p>If <var>selectorArg</var> is neither <code>null</code> nor
-                  a valid <a>selector</a>, return a promise rejected with a
-                  <code>TypeError</code>.</p>
+                  <p>Let <var>connection</var> be the
+                  <code><a>RTCPeerConnection</a></code> object on which
+                  the method was invoked.</p>
+                </li>
+                <li>
+                 <p>If <var>selectorArg</var> is neither <code>null</code> nor
+                 a valid <a>MediaStreamTrack</a>, throw a <code>TypeError</code>
+                 and abort these steps.</p>
```

Comment:
It's indistinguishable from a rejection [thanks to WebIDL](https://www.w3.org/2001/tag/doc/promises-guide#webidl-promise-return-values). That said, looking through this doc I see we mostly (*) use reject language, so I'll change it to be consistent.

*) [getIdentityAssertion](http://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-getidentityassertion) returns a promise, and throws as part of its algorithm. This returns a rejected promise rather than throwing in the caller's JS.